docs/guides/classical-feedforward-and-control-flow.ipynb

```diff
@@ -202,196 +190,194 @@
-    "    circuit.x(q0)\n",
-    "circuit.measure(q0, c0)\n",
-    "\n",
-    "circuit.draw(\"mpl\")\n",
-    "\n",
-    "# example output counts: {'1': 1024}"
+    "qc.draw(output=\"mpl\", style=\"iqp\", idle_wires=False, fold=-1)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "6f1050b0-58a5-46e1-95c6-980db3f09c00",
+   "id": "0378a45f-7a28-48bc-993d-0e4308d38bfc",
    "metadata": {},
    "source": [
-    "## While loop\n",
-    "\n",
-    "A while loop is used to repeat instructions while some condition is satisfied.\n",
-    "\n",
-    "In the example below, we apply Hadamard gates to two qubits and measure them. Then, we create a while loop that repeats this procedure while the measurement outcome is 11. As a result, the final measurement should never be 11, with the remaining possibilities appearing with approximately equal frequency."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 7,
-   "id": "95795aa7-9d4d-4c35-ba0e-48e6ba4f41b9",
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<Image src=\"/docs/images/guides/classical-feedforward-and-control-flow/extracted-outputs/95795aa7-9d4d-4c35-ba0e-48e6ba4f41b9-0.svg\" alt=\"Output of the previous code cell\" />"
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "qubits = QuantumRegister(2)\n",
-    "clbits = ClassicalRegister(2)\n",
-    "circuit = QuantumCircuit(qubits, clbits)\n",
-    "\n",
-    "q0, q1 = qubits\n",
-    "c0, c1 = clbits\n",
-    "\n",
-    "circuit.h([q0, q1])\n",
-    "circuit.measure(q0, c0)\n",
-    "circuit.measure(q1, c1)\n",
-    "with circuit.while_loop((clbits, 0b11)):\n",
-    "    circuit.h([q0, q1])\n",
-    "    circuit.measure(q0, c0)\n",
-    "    circuit.measure(q1, c1)\n",
-    "\n",
-    "circuit.draw(\"mpl\")\n",
-    "\n",
-    "# example output counts: {'01': 334, '10': 368, '00': 322}"
+    "## Limitations\n",
+    "\n",
+    "Be aware of the following constraints when running dynamic circuits in Qiskit Runtime.  Note that as new features and improvements are added throughout the coming year, some of these limitations will be lifted.\n",
+    "\n",
+    "- There is a limit of 2400 broadcasted bits in a job (not a circuit). Note that multiple `if` statements that operate on the same classic register is considered one broadcast.  For example, the following code broadcasts three bits:\n",
+    "   ```python\n",
+    "   bit[32] c;\n",
+    "\n",
+    "   if (c < 4) ...\n",
+    "   if (c < 5) ...\n",
+    "\n",
+    "   c[0] = measure $0;\n",
+    "   c[1] = measure $1;\n",
+    "   c[2] = measure $2;\n",
+    "   ```\n",
+    "- Operands in arithmetic and bitshift operations must be resolvable at compile time. For example, `expr.add(5, 2)` is valid, but `expr.add(cr, 2)` is not because the content of `cr` (which refers to a `ClassicalRegister`) is not known until execution time.\n",
```

Comment:
Should we also add that `cr` does not have a well-defined arithmetic type? i.e. I don't know whether to consider it a `uint[32]`, an `int[32]`, or something else.

Reply:
Qiskit's classical `expr` system lifts `cr` to be of our type `types.Uint(n)`, which is equivalent to OQ3's `uint[n]`.  For historical reasons (predating the `qiskit.circuit.classical` module) in OQ3 we declare the type of registers as `bit[n]`, which now I say it probably means our OQ3 output of arithmetic and comparative operators on registers is not strictly well typed (but it's well typed from within Qiskit itself).